src/tests/topp/FeatureFinderIdentification_1_output.featureXML

```diff
@@ -1,549 +1,454 @@
+			<ProteinHit id="PH_4" accession="P02769|ALBU_BOVIN" score="0" sequence="">
 				<UserParam type="float" name="OMSSA_score" value="0.0"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
-			<ProteinHit id="PH_5" accession="tr|A9GV08|A9GV08_SORC5" score="0" sequence="">
+			<ProteinHit id="PH_5" accession="Q14525|KT33B_HUMAN" score="0" sequence="">
 				<UserParam type="float" name="OMSSA_score" value="0.0"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
-			<ProteinHit id="PH_6" accession="Q15323|K1H1_HUMAN" score="0" sequence="">
+			<ProteinHit id="PH_6" accession="Q14532|K1H2_HUMAN" score="0" sequence="">
 				<UserParam type="float" name="OMSSA_score" value="0.0"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
-			<ProteinHit id="PH_7" accession="Q14532|K1H2_HUMAN" score="0" sequence="">
+			<ProteinHit id="PH_7" accession="Q15323|K1H1_HUMAN" score="0" sequence="">
 				<UserParam type="float" name="OMSSA_score" value="0.0"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
 			<ProteinHit id="PH_8" accession="Q92764|KRT35_HUMAN" score="0" sequence="">
 				<UserParam type="float" name="OMSSA_score" value="0.0"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
-			<ProteinHit id="PH_9" accession="O76013|KRT36_HUMAN" score="0" sequence="">
+			<ProteinHit id="PH_9" accession="tr|A9FUD7|A9FUD7_SORC5" score="0" sequence="">
 				<UserParam type="float" name="OMSSA_score" value="0.0"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
-			<ProteinHit id="PH_10" accession="O76014|KRT37_HUMAN" score="0" sequence="">
+			<ProteinHit id="PH_10" accession="tr|A9G0B6|A9G0B6_SORC5" score="0" sequence="">
 				<UserParam type="float" name="OMSSA_score" value="0.0"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
-			<ProteinHit id="PH_11" accession="O76015|KRT38_HUMAN" score="0" sequence="">
+			<ProteinHit id="PH_11" accession="tr|A9G8G1|A9G8G1_SORC5" score="0" sequence="">
 				<UserParam type="float" name="OMSSA_score" value="0.0"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
-			<ProteinHit id="PH_12" accession="Q14525|KT33B_HUMAN" score="0" sequence="">
+			<ProteinHit id="PH_12" accession="tr|A9GID7|A9GID7_SORC5" score="0" sequence="">
 				<UserParam type="float" name="OMSSA_score" value="0.0"/>
+				<UserParam type="string" name="target_decoy" value="target"/>
 			</ProteinHit>
-			<ProteinHit id="PH_13" accession="P00761|TRYP_PIG" score="0" sequence="">
-				<UserParam type="float" name="OMSSA_score" value="0.0"/>
-			</ProteinHit>
-			<ProteinHit id="PH_14" accession="P02769|ALBU_BOVIN" score="0" sequence="">
+			<ProteinHit id="PH_13" accession="tr|A9GN44|A9GN44_SORC5" score="0" sequence="">
```

Comment:
I think we should focus on the FFID tests first. Not even here, the numbers are the same. Was something changed in the ProteinHit Comparator??? They are sorted completely in the opposite direction. Pretty annoying for comparison.

Reply:
agree. first these issues

Reply:
@hendrikweisser Do you have any idea where that could go missing?

Reply:
@jpfeuffer I am not sure about changes in the ProteinHit Comparator. Could you give me a pointer where to look?

The problem is that normal `ProteinIdentification::sort()` does not do anything, since it will sort by score (0).

Reply:
It should also sort by accession

Reply:
See comment below.